ibm/ibm.go

```diff
@@ -109,6 +109,10 @@ type Provider struct {
 	IamEndpointOverride string `gcfg:"iamEndpointOverride"`
 	// Optional: Resource Manager endpoint override URL
 	RmEndpointOverride string `gcfg:"rmEndpointOverride"`
+	// Optional: PowerVS endpoint override URL
+	PowerEndpointOverride string `gcfg:"powerEndpointOverride"`
```

Comment:
Any reason for `PowerEndpointOverride` instead of `PowerVSEndpointOverride` ?

Reply:
No specific reason, I just named it, I can update it to PowerVSEndpointOverride

Reply:
Updated it to `PowerVSEndpointOverride`